srcpkgs/lsd/template

```diff
@@ -3,7 +3,7 @@ pkgname=lsd
 version=1.1.5
 revision=1
```

Comment:
revsion needs to be incremented so it will rebuild

Reply:
I noticed that in the guidelines *after* I made the pull request. I know it is possible to add more commits to a PR, but could not find how in the github site.

Note: I am familiar with git, but not with github. I made the edit, commit and PR all via the github website.
Sorry for the inconvenience.

Reply:
you can do `git reset --soft HEAD~2` to get back to before your commits, then recommit with `git commit`

in the future, you can use `git commit --amend` to edit the message and contents of the commit

both of these actions require `git push --force` to update the PR